crates/sui-graphql-client/src/lib.rs

```diff
@@ -643,7 +646,7 @@ impl Client {
     /// provided, it will use the last known checkpoint id.
     pub async fn checkpoint(
         &self,
-        digest: Option<Digest>,
+        digest: Option<CheckpointContentsDigest>,
```

Comment:
this probably should be CheckpointDigest and not Contents given the return type here

Reply:
You should take a look at the checkpoint types in the sdk types crate to familiarize yourself for what the differences are. The CheckpointDigest is the digest of a CheckpointSummary while the CheckpointContentsDigest is the digest of the "contents" or the CheckpointContents type which contains the list of txn digests and user signatures

Reply:
That's what I gathered as well, but I think then we're missing a `digest` field in the `CheckpointSummary` type.

Reply:
Actually, I am confused.